### PR TITLE
issue-08

### PR DESCRIPTION
**Pull request by mfaisalkhatri**
Unit tests added.

### CODE REVIEW

File: src/test/java/com/github/wasiqb/coteafs/selenium/pages/DropdownPage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	 * @since Jul 19, 2020
+	 *
+	 */
+	public enum DropdownKeys implements ElementKey {
+
+		/**
+		 * @author Faisal Khatri
+		 * @since Jul 19, 2020
+		 */
+		OPTION (" ");
```
Comment (WasiqB) [inline src/test/java/com/github/wasiqb/coteafs/selenium/pages/DropdownPage.java:27]: Why the key is empty? You need to specify proper key.

File: src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * @author Faisal Khatri
+ * @since Jul 19, 2020
+ *
+ */
+public class CheckboxPageAction extends AbstractPageAction <CheckboxPageAction> {
+
+	@Override
+	public void perform () {
+		final CheckboxPage checkbox = new CheckboxPage ();
+		if (value (CheckboxPageKeys.CHECK).equals ("check")) {
```
Comment (WasiqB) [inline src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java:19]: Switch case can be used.

File: src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			checkbox.checkBoxOne ()
+				.verifySelected ()
+				.isFalse ();
+			checkbox.checkBoxTwo ()
+				.verifySelected ()
+				.isFalse ();
+
+		}
+		else {
+			Assert.fail ("Test failed as invalid  test data is not supplied!");
```
Comment (WasiqB) [inline src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java:44]: Use class `ErrorUtil` from coteafs-error library to fail the test.

File: testng.xml [hunk 4]
```diff
@@ -29,11 +29,13 @@
...
                 <methods>
                     <include name="testLogin"/>
+                    <include name="testCheckboxes"/>
+                    <include name="testDropdownBox"/>
                 </methods>
             </class>
         </classes>
     </test>
-    <test name="Test Grid">
+    <test name="Test Grid" enabled = "true">
```
Comment (WasiqB) [inline testng.xml:38]: `enabled` attribute is not required.

File: testng.xml [hunk 5]
```diff
@@ -44,7 +46,7 @@
             </class>
         </classes>
     </test>
-    <test name="Test BrowserStack">
+    <test name="Test BrowserStack" enabled = "true">
```
Comment (WasiqB) [inline testng.xml:49]: `enabled` attribute is not required.

**Comment by WasiqB** [comment]
Fix the checkstyle issues.

File: src/test/java/com/github/wasiqb/coteafs/selenium/pages/DropdownPage.java (hunk 1 above)
Comment (mfaisalkhatri) [reply src/test/java/com/github/wasiqb/coteafs/selenium/pages/DropdownPage.java:27]: Updated.

File: src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java (hunk 2 above)
Comment (mfaisalkhatri) [reply src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java:19]: Updated.

File: src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java (hunk 3 above)
Comment (mfaisalkhatri) [reply src/test/java/com/github/wasiqb/coteafs/selenium/pages/action/CheckboxPageAction.java:44]: Removed this as it is not required.

File: testng.xml (hunk 4 above)
Comment (mfaisalkhatri) [reply testng.xml:38]: Removed.

File: testng.xml (hunk 5 above)
Comment (mfaisalkhatri) [reply testng.xml:49]: Removed.

**Pull request merged**